### DISCUSSION TITLE
Tablet Size

### DISCUSSION THREAD

**Discussion by mega-hz** [ulic75/power-flow-card#153] [Q&A]
Hi, i want to use this on a Tablet on a wall, in portrait direction, but the Lines from grid to home are too long.
Is there a place in the .js file where i one can change this?

**Comment by ulic75** [comment]
There isn't a configuration, but you can modify the svg manually if you want.

**Comment by mega-hz** [comment]
svg? sorry, i only have the .js file in HA.
I am no JS programmer, so i dont know where to change.
could you help me please?

**Reply by ulic75** [reply]
The "images" in the card are vector graphics made using svg, all of which is in the js file.  I'm afraid what you're asking for is not a simple task.

**Comment by mega-hz** [comment]
it would be ok if the width would be as in the HA Energy view, but this one is too wide.
